src/quo2/components/list_items/preview_list.cljs

```diff
@@ -0,0 +1,115 @@
+(ns quo2.components.list-items.preview-list
+  (:require [quo.react-native :as rn]
+            [status-im.i18n.i18n :as i18n]
+            [quo2.foundations.colors :as colors]
+            [quo2.components.icon :as quo2.icons]
+            [quo2.components.avatars.user-avatar :as user-avatar]
+            [quo2.components.markdown.text :as quo2.text]))
+
+(def params {32 {:border-radius {:circular 16 :rounded  10}
+                 :hole-radius   {:circular 18 :rounded  12}
+                 :margin-left   -8
+                 :hole-size     36
+                 :hole-x        22
+                 :hole-y        -2}
+             24 {:border-radius {:circular 12 :rounded  8}
+                 :hole-radius   {:circular 13 :rounded  9}
+                 :margin-left   -4
+                 :hole-size     26
+                 :hole-x        19
+                 :hole-y        -1}
+             16 {:border-radius {:circular 8 :rounded  8}
+                 :hole-radius   {:circular 9 :rounded  9}
+                 :margin-left   -4
+                 :hole-size     18
+                 :hole-x        11
+                 :hole-y        -1}})
+
+;; TODO - Add avatar components for other types once implemented
+(defn avatar [item type size]
+  (case type
+    :user [user-avatar/user-avatar
+           (merge {:ring?             false
+                   :status-indicator? false
+                   :size              (case size 32 :small 24 :xs 16 :xxxs)}
```

Comment:
I don't know why we are using names like this for sizes. Figma doesn't use them and they are not consistent.
For example for [icon avatar](https://github.com/status-im/status-mobile/blob/3fc5c898658be7afa23d4d43a042777c1dddd629/src/quo2/components/avatars/icon_avatar.cljs#L8) `:big` is 48, while for [user avatar](https://github.com/status-im/status-mobile/blob/3fc5c898658be7afa23d4d43a042777c1dddd629/src/quo2/components/avatars/user_avatar.cljs#L10)  `:big` is 80.
Please can we use just size literals for sizes, as I am using in [preview_list](https://github.com/status-im/status-mobile/pull/13830/files#diff-1f6e7fdc52d1ca278044e0bb84505546cc8e91b1a17c8325c820fc99db63f97d) component instead of naming them
cc: @ibrkhalil

Reply:
So, I can just pass `:size 32` instead of figuring out its name.

Reply:
You're right .. We can pass the value instead .. that'll make it better